### PR TITLE
T223709: Add new config variable to force default user usage

### PR DESCRIPTION
**Pull request by gandie**

### CODE REVIEW

File: perfact/zodbsync/zodbsync.py [hunk 1]
```diff
@@ -138,7 +140,9 @@ def mod_write(data, parent=None, obj_id=None, override=False, root=None,
     d = dict(data)
     meta_type = d['type']
 
-    if default_owner is not None and 'owner' not in d:
+    no_owner_given = 'owner' not in d
+
+    if (default_owner and no_owner_given) or (force_default_owner):
```
Comment (viktordick) [inline perfact/zodbsync/zodbsync.py:145]: I guess giving no `default_owner` but setting `force_default_owner` is not supported and does not need to give a sensible result.

File: perfact/zodbsync/tests/test_sync.py [hunk 2]
```diff
@@ -1215,3 +1215,73 @@ def test_no_meta_file(self):
...
+
+        expected_owner = (['acl_users'], self.runner.sync.default_owner)
+
+        assert self.app.newfolder._owner == expected_owner
+
+        # second test: owner from zope read to meta file
+        with self.runner.sync.tm:
+            self.app.manage_addProduct['OFSP'].manage_addFolder(id='another')
+
+        self.app.another._owner = 'Somebody'
```
Comment (viktordick) [inline perfact/zodbsync/tests/test_sync.py:1247]: This is not a valid owner (it should be a tuple), but I guess this does not really matter here.

**Review by viktordick — APPROVED** [review]
LGTM!

**Comment by gandie** [comment]
Adjusted the owner in tests ( to be valid ! ) and clarified conditions.

**Pull request merged**